.github/workflows/ci-test.yml

```diff
@@ -212,6 +212,8 @@ jobs:
       - prepare
     steps:
       - uses: actions/checkout@v2.4.0
+      - name: install rust toolchain
```

Comment:
Are these changes necessary for this extension to work?

Reply:
Yes, this plugin depends on the move-analyzer binary, in order to verify that the plugin is working properly, we need to compile and install the move-analyzer binary first。

The install code:  https://github.com/yubing744/starcoin-move/blob/0be56e1ec54dc67c0ae7ddd0eb8d8f7f0c58abc2/language/move-analyzer/editors/code/package.json#L89